### ISSUE TITLE
Images des albums déformées

### ISSUE THREAD

**Issue by Aisyk** [musique-libre/dogmazic.net#10]
Bonjour,

Petit "bug" graphique, les images des albums (typiquement des images carrées), sont déformées.
https://framapic.org/HpJrz31KQhxY/C5p0mdHq1wXZ.png

**Comment by UncleBone** [comment]
Oui, le sujet a été évoqué sur le forum. C'est une modif de fufroma qui rentre en conflit avec le CSS. Je corrige ça dès que j'ai accès au serveur, à moins que fufroma ne s'en occupe avant.
